vsc/model/vsc_ast.py

```diff
@@ -23,63 +26,63 @@ class Error:
 class Method:
     name: str
     description: Optional[str] = None
-    error: Optional[List[Error]] = field(default_factory=lambda: [])
-    input: Optional[List[Argument]] = field(default_factory=lambda: [])
-    output: Optional[List[Argument]] = field(default_factory=lambda: [])
+    error: List[Error] = field(default_factory=EmptyList)
+    input: List[Argument] = field(default_factory=EmptyList)
+    output: List[Argument] = field(default_factory=EmptyList)
 
 
 @dataclass
 class Event:
     name: str
-    description: Optional[str] = None
-    input: Optional[List[Argument]] = field(default_factory=lambda: [])
+    description: str = None
```

Comment:
Some have `str = None`, some have `str = str()`
Also here the description is an _optional_ thing.

Reply:
When you have a field `description: Optional[str] = None` in the template without checking the current value, you would have `None` as a string. So, for some of the fields, like description, I added `str()` to not have None in the template output. 
This is also a more philosophical question.